frame/staking-rewards/src/lib.rs

```diff
@@ -85,11 +132,150 @@ pub mod pallet {
 		#[pallet::constant]
 		type PalletId: Get<PalletId>;
 
+		/// Maximum number of staking duration presets allowed.
+		#[pallet::constant]
+		type MaxStakingDurationPresets: Get<u32>;
+
+		/// Maximum number of reward configurations per pool.
+		#[pallet::constant]
+		type MaxRewardConfigsPerPool: Get<u32> + TypeInfo;
```

Comment:
u8

Reply:
u32 is required as per the bounds on `BoundedBTreeMap` impls